app/jobs/update_vacancy_spreadsheet_job.rb

```diff
@@ -0,0 +1,22 @@
+require 'spreadsheet_writer'
+
+class UpdateVacancySpreadsheetJob < ApplicationJob
+  queue_as :update_vacancy_spreadsheet
+
+  PUBLISHED_VACANCY_SPREADSHEET_ID = ENV['PUBLISHED_VACANCY_SPREADSHEET_ID']
```

Comment:
Can this be moved in an initialiser, so all relevant configuration is in one place.

[The google_keys initializer]( https://github.com/dxw/teacher-vacancy-service/blob/develop/config/initializers/google_keys.rb) looks like a good candidate as it's only used by the `SpreadsheetWriter`. The initializer should probably be renamed to reflect that as well.